### PR TITLE
Added a badge for date information of an event. (#88)

### PR DESCRIPTION
**Pull request by Agreon**

### CODE REVIEW

**Comment by Agreon** [comment]
As there is no event this week that has inverted colors here is a screenshot: 

Selection_013

File: templates/macros/event.njk [hunk 1]
```diff
@@ -6,6 +6,19 @@
   {% endif %}
 {% endmacro %}
 
+{% macro invert(twitterInfo) %}
+  {% if twitterInfo.hashtag %}
+    invert
+  {% endif %}
+{% endmacro %}
+
+{% macro dateInfo(isToday, isThisWeek, twitterInfo) %}
```
[the remark this answers is not in the document]
Comment (hell03610) [reply templates/macros/event.njk]: But sometimes it catches me as in https://github.com/VLCTechHub/VLCTechHub-site/blob/5cff9d2aecbd5931fa6395934687f40b592f8c2d/templates/macros/event.njk#L1 :grinning:

File: templates/macros/event.njk [hunk 2]
```diff
@@ -6,6 +6,19 @@
...
+  {% if isToday %}
+    <div class="date-badge {{invert(twitterInfo)}}">Hoy</div>
```
Comment (hell03610) [inline templates/macros/event.njk]: Just to let you know, a pattern I do in other repos is to return all the class names in one method, I've found that is more clear, as in:
```
<div class="{{dateBadgeClassNames(twitterInfo)}}">Hoy</div>
...
```

File: templates/macros/event.njk (hunk 1 above)
[the remark this answers is not in the document]
Comment (Agreon) [reply templates/macros/event.njk]: I tried to stay consistent :P

[the remark this answers is not in the document]
Comment (Agreon) [reply templates/macros/event.njk]: Seems legit.

[the remark this answers is not in the document]
Comment (Agreon) [reply templates/macros/event.njk]: I tried to stay consistent :P

File: templates/macros/event.njk (hunk 2 above)
Comment (Agreon) [reply templates/macros/event.njk]: Seems like a good idea :)

**Comment by Agreon** [comment]
No Problem. I really have fun working on this site. It seems like you have a nice community in Valencia.

**Comment by hell03610** [comment]
Hahaha, that was fast! Let me know if you happen to come around ^_^ Good night!

fast keyboard

**Pull request merged**